lib/sessions.js

```diff
@@ -44,18 +44,16 @@ module.exports = (app, config) => {
   app.use(cookieParser(config.session.secret, {
     path: '/',
     httpOnly: true,
-    secure: true
+    secure: config.protocol === 'https'
   }));
 
-  const sessionOpts = Object.assign({}, {
+  const sessionOpts = Object.assign({
     store,
-    cookie: {secure: true},
-    key: '',
```

Comment:
I rather like the idea of having a custom cookie name. Should we default it something other than '', but make it configurable?